src/main/java/org/prebid/server/handler/SetuidHandler.java

```diff
@@ -121,6 +132,18 @@ private static UsersyncMethodType preferredUserSyncType(Usersyncer usersyncer) {
                 .get(); // when usersyncer is present, it will contain at least one method
     }
 
+    private static void validateUsersyncers(List<Usersyncer> usersyncers) {
+        final List<String> cookieFamilyNames = usersyncers.stream()
+                .map(Usersyncer::getCookieFamilyName)
+                .toList();
+        cookieFamilyNames.stream()
+                .filter(name -> Collections.frequency(cookieFamilyNames, name) > 1)
+                .findAny()
+                .ifPresent(name -> {
+                    throw new IllegalArgumentException("Duplicated \"cookie-family-name\" found, value: " + name);
```

Comment:
You will only print the first one you find. But there could be many misconfigured elements. Minor, but I'd prefer to have a list of duplications.

Reply:
Got it. But in this case separated value needed anyway.